packages/web/src/components/range/RangeInput.js

```diff
@@ -185,6 +185,7 @@ RangeInput.propTypes = {
 	style: types.style,
 	themePreset: types.themePreset,
 	componentId: types.stringRequired,
+	includeNullValues: types.bool,
```

Comment:
@ShahAnuj2610 Are we missing the query generation here?

Reply:
can you elaborate?

Reply:
Ok, I think it is not needed.